api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -3033,6 +3039,20 @@ private void enableCohortBuilder() {
 
     }
 
+    private void enableHaitiHIVLink() {
+        extensions.add(extension(
+                CustomAppLoaderConstants.Extensions.HAITI_HIV_EMR_DASHBOARD_LINK,
+                "pihcore.hiv.viewPatientInHivEmr",
+                "icon-external-link",
+                "link",
+                "https://haitihivtest.pih-emr.org/openmrs/pihcore/patient.page?patientId={{identifier}}&identifierType=" + ZlConfigConstants.PATIENTIDENTIFIERTYPE_HIVEMRV1_UUID + "&dashboard="+ PihEmrConfigConstants.PROGRAM_HIV_UUID,
```

Comment:
I'll test that this works with the "haitihivtest" server on HUM-CI, and then change this to "haiti" and disable on HUM-CI.